sdk/modelparser/azure-digitaltwins-parser/src/main/java/com/azure/digitaltwins/parser/implementation/codegen/JavaType.java

```diff
@@ -16,16 +16,16 @@
  */
 public class JavaType extends JavaDeclaration implements JavaFile {
 
-    private ClientLogger logger = new ClientLogger(JavaType.class);
-    private String typeName;
+    private final ClientLogger logger = new ClientLogger(JavaType.class);
```

Comment:
does this need to be static as well?

Reply:
yap, I will change them all. good catch.